src/extension/src/utils/vscodeUI.ts

```diff
@@ -33,7 +33,7 @@ export class VSCodeUI extends WizardServant {
   async promptUsersToResetPages(message: any) {
     if (
       message.payload.pagesLength > 0 &&
-      message.text === PAYLOAD_MESSAGES_TEXT.SWITCH_FRAMEWORKS_TEXT
+      message.text === PAYLOAD_MESSAGES_TEXT.RESET_PAGES_TEXT
```

Comment:
why? is this just outstanding from before?

Reply:
So, now we no longer need to reset pages on framework change. We reset the pages only if RESET_PAGES_TEXT is the payload for resetting pages. As of now, we could remove this text altogether, but it is good to have to distinguish between different commands if we want different behavior in future.